Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/types.go

```diff
@@ -233,6 +235,8 @@ type PersistentVolumeSource struct {
 	// ISCSIVolumeSource represents an ISCSI resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	ISCSI *ISCSIVolumeSource `json:"iscsi"`
```

Comment:
Elsewhere in this PR, `omitempty` is added to the json tag for ISCSI, but not here.